mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/jobcluster/job/JobActor.java

```diff
@@ -1586,7 +1586,7 @@ private ScheduleRequest createSchedulingRequest(
                                 mantisJobMetaData.getMinRuntimeSecs()
                         ),
                         mantisJobMetaData.getSla().orElse(new JobSla.Builder().build()).getDurationType(),
-                        stageMetadata.getMachineDefinition(),
+                        AllocationConstraints.of(stageMetadata.getMachineDefinition(), mantisJobMetaData.getJobDefinition().getAssignmentAttributes()),
```

Comment:
nit: stageMetadata.getSchedulingConstraints() would be cleaner.

Reply:
good feedback but I'm trying to avoid changing too many things at once.